### ISSUE TITLE
[Dropdown] Selected action ignored on init

### ISSUE THREAD

**Issue by TivoSoho** [Semantic-Org/Semantic-UI#6783]
### Steps
1. Make a dropdown
2. Set the dropdown action to "nothing"

### Expected Result
It does nothing on first init. With action nothing I want to be in full control of how it reacts to selected values. I have set it to nothing since I want to keep them visible and unbolded and otherwise handle the clicking/choosing items myself.

### Actual Result
It activates and hides the selected values on init. 

### Version
2.4.2.

### Testcase
https://jsfiddle.net/mwr8y9be/

**Comment by lubber-de** [comment]
You were missing some HTML markup. I adjusted your fiddle and it does nothing now on selection as expected
https://jsfiddle.net/2mzotund/

**Comment by TivoSoho** [comment]
No, my dropdown is exactly right. I do NOT have a selection dropdown. That is something completely different.

**Comment by hammy2899** [comment]
@TivoSoho I think this is what your looking for https://jsfiddle.net/63r097bp/

**Comment by TivoSoho** [comment]
Thank you for the effort but nope. As you can clearly see in your own demo - it has the same bug. It does something - it sets a label value. The action is nothing. So far the only workaround is the "onShow" function to remove those "filtered" and "active" classes, but nonetheless I consider it a bug until someone comes with a more clear way to stop the init function from doing too much.

**Comment by lubber-de** [comment]
@TivoSoho In my above example https://jsfiddle.net/2mzotund/ it does not set the label, when i click on any entry, it does nothing. 🤔 
I am probably still misunderstanding ?

**Comment by TivoSoho** [comment]
@lubber-de Please look at my example. I don't have a selection dropdown, as explained. It's not a "select".

**Comment by lubber-de** [comment]
@TivoSoho Ok, got it, you changed the fiddle to implement a button. Sorry, missed that.
Ok, i adjusted your fiddle to remove the initial value "selected", so both of the dropdown items are not seen without any action when clicked
https://jsfiddle.net/7a6pco3s/
As far as i understood now, you expect even the "hard" defined value of the input field to be ignored for preselection.
At the moment the "action" setting is only used on the item.click event, so this has to be adjusted to fullfill your request.

**Comment by TivoSoho** [comment]
@lubber-de Yeah, but I am using the input.
